OsmAnd/res/values/strings.xml

```diff
@@ -146,18 +146,18 @@
     <string name="nautical_depth_points">Nautical depth points</string>
     <string name="depth_contours">Depth contours</string>
     <string name="worldwide_maps">Worldwide maps</string>
-    <string name="map_widget_target_distance_desc">Shows the distance between camera and target location.</string>
+    <string name="map_widget_target_distance_desc">Shows the distance from the camera to the target location.</string>
```

Comment:
"From ... to" sounds clumsier?

Reply:
From the user point of view, the camera is pointed, thus it is less complex to start there was my thinking.